types/bpk-component-breakpoint.d.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+declare module 'bpk-component-breakpoint';
```

Comment:
Discussing if:

- these should be in /@types/
- all the bpk ones should be in a single file

in https://github.com/Skyscanner/backpack/pull/2535

👀

Reply:
Whatever you think's best, let us know if you want us to change it :)